src/App.css

```
@@ -43,6 +56,10 @@ body{
   background: #0D0F16;
 }
 
+a{
+  text-decoration: none;
+}
+
 section {
   display: flex;
```

```suggestion
a {
  text-decoration: none;
}

section {
  display: flex;
```

please check your prettier